tutorials/dataframe/df106_HiggsToFourLeptons_withVariations.py

```diff
@@ -118,53 +130,115 @@
     histos.append(df.Filter(f"sample_category == \"{sample_category}\"").Histo1D(ROOT.RDF.TH1DModel(f"{sample_category}", "m4l", 24, 80, 170), "m4l", "weight"))
 
 
-h_data = histos[0].GetValue()
-h_higgs = histos[1].GetValue()
-h_zz = histos[2].GetValue()
-h_other = histos[3].GetValue()
+# Evaluate the systematic uncertainty
 
-# Apply MC correction for ZZ due to missing gg->ZZ process
-h_zz.Scale(1.3) 
+# The systematic uncertainty in this analysis is the MC scale factor uncertainty that depends on lepton 
+# kinematics such as pT or pseudorapidity. 
+# Muons uncertainties are negligible, as stated in https://atlas.web.cern.ch/Atlas/GROUPS/PHYSICS/PAPERS/MUON-2018-03/. 
+# Electrons uncertainties are evaluated based on the plots available in https://doi.org/10.48550/arXiv.1908.00005. 
+# The uncertainties are linearly interpolated to cover a range of pT values covered by the analysis. 
 
-# Create the plot
+# Create a VaryHelper to interpolate the available data.
+ROOT.gInterpreter.Declare("""   
+#include <Math/Interpolator.h>                                   
```

Comment:
```suggestion
```
I think you don't need this in the interpreter, since all the ROOT headers are included on startup.